### ISSUE TITLE
ERROR: muscle on X failed

### ISSUE THREAD

**Issue by cirodri1** [molbiodiv/bcgTree#26]
Dear all,

I was trying to use the graphical user interface to run a "test" for only 4 sequences as shown by the screenshot, and every time I run it it shows an error saying "ERROR: muscle on (some ID-it changes) failed.

An example of what I mean is demonstrated below:

It shows like the process starts fine:
<img width="886" alt="screen shot 2018-05-02 at 4 32 48 pm" src="https://user-images.githubusercontent.com/36938244/39554262-adcb11ee-4e26-11e8-906c-e6f3c8948637.png">

But then it shows the ERROR: and the end:
<img width="1808" alt="screen shot 2018-05-02 at 4 27 48 pm" src="https://user-images.githubusercontent.com/36938244/39554267-b812a16c-4e26-11e8-95c0-a3113aa4ff36.png">

<img width="893" alt="screen shot 2018-05-02 at 4 36 04 pm" src="https://user-images.githubusercontent.com/36938244/39554317-fbba929e-4e26-11e8-858b-52e921ed71c8.png">

Any ideas on what might be causing the issue?

I am very new at bioinformatics so I apologize if this is a silly question.

Thanks for your help!

**Comment by iimog** [comment]
Hi @cirodri1,

it appears, that you supply only `/Users/cynthiarodriguez/` as path to `muscle`. This is probably only the directory in which `muscle` is located. Could you please try providing the full path to the `muscle` binary (for example `/users/cynthiarodriguez/muscle3.8.31_macppc`).
Unfortunately our `check` button only checks for the existence of the path you provide and does not check if it is the correct program. This is something that we should improve. Thanks for reporting this issue and please let me know if my suggestion solves your problem.

Best,
Markus

**Comment by cirodri1** [comment]
Dear Markus,

Thanks for the fast and helpful reply. I changed the path to /users/cynthiarodriguez/muscle and it fixed the problem!
<img width="770" alt="screen shot 2018-05-03 at 10 11 08 am" src="https://user-images.githubusercontent.com/36938244/39592496-c8586d5a-4ebb-11e8-9b5d-9b3bb4310ad1.png">

Thank you very much for your help!

I have one more question:

when the process is done and I go to the output folder I am able to retrieve the "RAxML_bestTree.final" and it order to visualize it I opened it in the iTOL online tool as shown below:
<img width="1098" alt="screen shot 2018-05-03 at 10 20 34 am" src="https://user-images.githubusercontent.com/36938244/39592568-ff3b5e40-4ebb-11e8-98c5-bf1a531cf233.png">

Now my question is how I can add the bootstrapping values to the branches of my trees?
These are the files available to me in the output file:
<img width="568" alt="screen shot 2018-05-03 at 10 26 28 am" src="https://user-images.githubusercontent.com/36938244/39592738-89fbc042-4ebc-11e8-8ea2-e65178c91387.png">
 and I tried opening the file called "RAxML_bootstraps.final" thinking it might show the boostrapping values but nothing showed up.

Thank you very much for your help again!

Best,

Cynthia
